CRM/Case/DAO/Case.php

```diff
@@ -260,7 +260,7 @@ public static function &fields() {
           'localizable' => 0,
           'html' => [
             'type' => 'Select Date',
-            'formatType' => 'activityDateTime',
+            'formatType' => '`activityDate',
```

Comment:
@mattwire looks like a stray ` in` '`activityDate',`

Reply:
Fixed thanks